lib/ServerFiles.php

```diff
@@ -199,4 +199,36 @@ public function deleteFile() {
 
 		return new Result();
 	}
+
+	/**
+	 * List file in the specified directory under the server root
+	 *'file' is the file to delete, including path from the server root
+	 * 'dir' is the directory to check, which may be inside other directories
+	 * e.g. 'apps2/myapp/appinfo'
+	 *
+	 * @return Result
+	 */
+	public function listFile() {
```

Comment:
```suggestion
	public function listFiles() {
```
Call this method `listFiles` ?

Reply:
Okay